data/web.py

```diff
@@ -0,0 +1,6 @@
+from community.git import get_org_name
+
+
+def web_url():
```

Comment:
this is not "web".  it is "webservices"

and refer to the `get_upstream_remote_url` discussion on li-boxuan 's patch